docs/installation-guide/quickstart.rst

```diff
@@ -8,14 +9,15 @@ the amount of resources required to implement the design.
 
 .. _MetalK8s: https://github.com/scality/metalk8s/
 .. _CentOS: https://www.centos.org
-.. _Kubernetes: https://kubernetes.io
+
 
 General Cluster Requirements
 ----------------------------
 Setting up a MetalK8s_ cluster quickly requires at least three machines
```

Comment:
Couldn't we start with one machine? even is 3 is recommended?

Reply:
I think I'll rework this part when explaining what the control plane is, but no, it's not a hard requirement.